docker/run.sh

```diff
@@ -39,17 +39,31 @@ PGPASSWORD=$DB_PASSWORD psql "sslmode=verify-ca sslrootcert=/root/.postgresql/ro
 -c "\copy (SELECT row_to_json(t) FROM (SELECT id,qid,caze_case_id as case_id FROM casev2.uac_qid_link where last_updated >= '$START_OF_PERIOD' and last_updated < '$END_OF_PERIOD') t) To '$QID_FILE';"
 
 
-echo "zipping uac_qid_link file"
+if [ -z "$DATAEXPORT_MI_BUCKET_NAME" ]
+then
+      echo "$DATAEXPORT_MI_BUCKET_NAME is empty"
```

Comment:
This `echo` is evaluating the variable so it'll print  "` is empty`"
```suggestion
      echo "DATAEXPORT_MI_BUCKET_NAME is not set"
```